src/CqlBuilderPanel/functionsSection/FunctionsSection.tsx

```diff
@@ -32,7 +98,15 @@ export default function FunctionsSection({
             handleApplyFunction={handleApplyFunction}
           />
         )}
-        {activeTab === "saved-functions" && <Functions />}
+        {activeTab === "saved-functions" && (
+          <Functions
+            canEdit={canEdit}
+            loading={loading}
+            functions={_.sortBy(functionLookups)}
```

Comment:
at this point list would already be sorted. I don't think we need to sort again.

Reply:
@adongare I removed it. Thanks!